### DISCUSSION TITLE
How to use flashbots with wagmi?

### DISCUSSION THREAD

**Discussion by cattingcat** [wevm/wagmi#720] [Q&A]
I tried to setup flashbots for wagmi and I heed to sign transaction and the send it to specific provider, so I know that it is impossible to do with metamask (because i doesn't support signing without sending) but WalletConnect does. So I used wagmi to get everything (provider/signer/connector and so on) and it just doesn't work for me. Could somebody share experience with it?

**Comment by nasa8x** [comment]
Wagmi is more efficient for react apps. If using flashbot, you should use [Hardhat](https://hardhat.org/docs) more optimally.

**Reply by cattingcat** [reply]
Well, I'm not sure about that. You should be able to use flashbots with any web3/web framework. Because it is just special provider to which you send signed transactions
